.github/workflows/unit-tests.yml

```
@@ -3,39 +3,18 @@ name: Unit tests
 on: [ pull_request ]
 
 jobs:
-  build_java_8:
+  build:
```

Could you modify the code to clarify its purpose?
```suggestion
linux-build-java-${{ matrix.version }}:
```